JEKScrollableSectionCollectionViewLayout.m

```diff
@@ -29,6 +29,7 @@ @interface JEKScrollableSectionInfo : NSObject
 @property (nonatomic, assign) CGSize headerSize;
 @property (nonatomic, assign) CGSize footerSize;
 @property (nonatomic, assign) BOOL needsLayout;
+@property (nonatomic, assign) BOOL useFlowLayout;
```

Comment:
Another nitpick where the more "Objective-C"-ish name would be `shouldUseFlowLayout`